### ISSUE TITLE
Tooltips fail WCAG 1.4.13 (AA) - cannot be hovered over or dismissed

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3463]
As per WCAG 1.4.13 (AA), tooltips need to be dismissible (when hovering over the glossary term, the user should be able to hit Esc and have the tool tip close, without moving their mouse. They also should be able to move their mouse onto the open tooltip without having it close. NRRD tooltips (glossary terms) fail both of these.

This problem was identified by a USWDS github discussion.

**Comment by lpgoldstein** [comment]
Upgrading
